packages/core/src/components/ButtonGroup/styles.ts

```diff
@@ -28,12 +28,23 @@ const styleSheet: StyleSheet = ({ unit }) => ({
   },
 
   cell_stacked: {
+    marginRight: 0,
     marginBottom: unit,
 
     ':last-of-type': {
       marginBottom: 0,
     },
   },
+
+  cell_stretched: {
+    flex: 1,
+
+    '@selectors': {
+      '> button, > a, > div, > span': {
```

Comment:
`> * { flexGrow: 1 }`?

Reply:
tried *, will try again

Reply:
flexGrow doesn't work because they're grandchildren of the flex